build/config/PublishData.json

```diff
@@ -21,6 +21,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
```

Comment:
@jaredpar So are we still uploading packages from this branch to the main feeds?? Or should we delete this entry now?